### PR TITLE
Task1 part1 layout structuring and slicing

### PR DESCRIPTION
**Pull request by abood454**
part 1

### CODE REVIEW

File: task_1/index.html [hunk 1]
```diff
@@ -1,15 +1,175 @@
...
+
+                </div>
+            </div>
+
+
+        </div>
 </body>
 <script src="./index.js"></script>
 
+
```
Comment (araraltawarah98) [inline task_1/index.html]: Remove spaces

File: task_1/index.html [hunk 2]
```diff
@@ -1,15 +1,58 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./style.css"><link>
-    <title>Document</title>
+    <title>Task 1</title>
+    <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
-    
+    <script src="./index.js">
```
Comment (laithyounes) [inline task_1/index.html:13]: scripts should be moved to the end of the body tag

File: task_1/index.html [hunk 3]
```diff
@@ -1,15 +1,175 @@
...
+        <h5>Our Lands</h5>
+        <h5>Our Suppliers</h5>
+        <h5>Products</h5>
+        <h5>Contact Us</h5>
+        <p>image</p>
+    </div>
+    <div id="Main">
+        <div id="Child1">
+            <div id="cont">
+
```
Comment (araraltawarah98) [inline task_1/index.html]: remove space

File: task_1/index.html [hunk 4]
```diff
@@ -1,15 +1,58 @@
...
+    <link rel="stylesheet" href="./style.css">
 </head>
+
 <body>
-    
+    <script src="./index.js">
+    </script>
+    <div class="wrapper">
+        <div class="header">
+            <!-- <ul>
```
Comment (laithyounes) [inline task_1/index.html:17]: please remove unnecessary comments, this might cause conflicts in the future

File: task_1/index.html [hunk 5]
```diff
@@ -1,15 +1,58 @@
...
+                <li>Contact Us</li>
+                <li>image</li>
+            </ul>-->
+        </div>
+        <div class="main">
+            <div class="hero-image"></div>
+            <div class="hero-banner">
+                <div class="text-banner">
+                    <p class="ProductFont">Be Active</p>
+                    <p class="OpenFont">Barnsley Brew Coffee</p>
```
Comment (laithyounes) [inline task_1/index.html:33]: I see multiple naming conventions please change this to kebab-case

File: task_1/index.html [hunk 6]
```diff
@@ -1,15 +1,58 @@
...
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+        </div>
+    </div>
 </body>
-<script src="./index.js"></script>
 
 </html>
```
Comment (laithyounes) [inline task_1/index.html:58]: please add newline at the end of the file
ref: https://stackoverflow.com/a/5813359

File: task_1/style.css [hunk 7]
```diff
@@ -1,3 +1,69 @@
...
+    width: 217px;
+}
+
+.grid-items {
+    border: 1px solid black;
+    display: grid;
+    grid-gap: 32px;
+    grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
+    padding-top: 3rem;
 }
```
Comment (laithyounes) [inline task_1/style.css:69]: please add newline at the end of the file
ref: https://stackoverflow.com/a/5813359

**Review by laithyounes — CHANGES_REQUESTED** [review]
please apply the requested changes, other than that LGTM

**Review by Mohsen-Awad — APPROVED** [review]
great work

**Review by araraltawarah98 — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/style.css [hunk 8]
```diff
@@ -1,180 +1,69 @@
-body {
-    background: linear-gradient(180deg, rgba(228, 239, 233, 0) 0%, #ECF8F1 6.25%);
+@font-face {
```
Comment (ashrafabuodeh) [inline task_1/style.css:1]: good job!

File: task_1/index.html [hunk 9]
```diff
@@ -54,5 +52,7 @@
         </div>
     </div>
 </body>
+<script src="./index.js">
```
Comment (laithyounes) [inline task_1/index.html:55]: this should be inside the body tag

**Review by laithyounes — CHANGES_REQUESTED** [review]

**Review by laithyounes — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

**Review by ashrafabuodeh — APPROVED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

**Pull request merged**